app/views/shop/_closed_shop.html.haml

```diff
@@ -0,0 +1,21 @@
+.row.closed_shop_header
+  .small-12.columns
+    .content{ "darker-background" => true }
```

Comment:
First I repeated some conditionals at the page-view element level to set the background darker but then I tried this directive approach and it looks shorter (and faster without duplicate conditionals).
Is there a better way of doing this I may be missing?